plugins/ocm/README.md

```diff
@@ -1,104 +1,114 @@
-
-   1. The information about your hub is provided purely by the OCM configuration. In order to use this configuration please add the following to your `app-config.yaml` file:
-
-      ```yaml
-      # app-config.yaml
-      catalog:
-        providers:
-          ocm:
-            env: # Key is reflected as provider ID. Defines and claims plugin instance ownership of entities
-              name: # Name that the hub cluster will assume in Backstage Catalog (in OCM this is always local-cluster which can be confusing)
-              url: # Url of the hub cluster API endpoint
-              serviceAccountToken: # Token used for querying data from the hub
-              skipTLSVerify: # Skip TLS certificate verification, defaults to false
-              caData: # Base64-encoded CA bundle in PEM format
-      ```
-
-   2. The hub configuration can be tied to the kubernetes plugin configuration by providing the name of the hub. This is useful when you already use a kubernetes plugin in your backstage instance. Please add following to your `app-config.yaml` file:
-
-      ```yaml
-      # app-config.yaml
-      kubernetes:
-        serviceLocatorMethod:
-          type: "multiTenant"
-        clusterLocatorMethods:
-          - type: "config"
-            clusters:
-              - name: <cluster-name>
-              ...
-
-      catalog:
-        providers:
-          ocm:
-            env: # Key is reflected as provider ID. Defines and claims plugin instance ownership of entities
-              kubernetesPluginRef: <cluster-name> # Match the cluster name in kubernetes plugin config
-      ```
-
-      Please consult the documentation to [Backstage Kubernetes plugin](https://backstage.io/docs/features/kubernetes/configuration#configuring-kubernetes-clusters) for details on its configuration. Hub cluster must be connected via Service Account.
-
-3. Create new plugin instance in `packages/backend/src/plugins/ocm.ts`:
+2. Configure the OCM backend plugin using one of the following configurations:
+
+   - The OCM configuration provides the information about your hub. To use the OCM configuration, add the following code to your `app-config.yaml` file:
+
+     ```yaml
+     # app-config.yaml
+     catalog:
+       providers:
+         ocm:
+           env: # Key is reflected as provider ID. Defines and claims plugin instance ownership of entities
+             name: # Name that the hub cluster will assume in Backstage Catalog (in OCM this is always local-cluster which can be confusing)
+             url: # Url of the hub cluster API endpoint
+             serviceAccountToken: # Token used for querying data from the hub
+             skipTLSVerify: # Skip TLS certificate verification, defaults to false
+             caData: # Base64-encoded CA bundle in PEM format
+     ```
+
+   - You can bind the hub configuration to the Kubernetes configuration by providing the name of the hub as follows:
```

Comment:
```suggestion
   - If you also have Backstage Kubernetes plugin installed and configured to connect to the Hub cluster, you can defer to this configuration by providing the name of the hub as follows. Please make sure Backstage uses a `ServiceAccount` token and was granted permissions described above.
```

See the previous review comment. Feel free to reword it. 🙂

Reply:
Done